i18n/locales/es.json

```diff
@@ -35,5 +35,6 @@
   "components.pagination.next": "Siguiente | ",
   "components.pagination.previous": "Anterior",
   "layouts.blogPost.author.byLine": "{author, select, null {} other {By {author}, }}",
-  "layouts.blogIndex.currentYear": "News from {year}"
+  "layouts.blogIndex.currentYear": "News from {year}",
+  "components.codeBox.copy": "{copied, select, true {copied}other {copy}}"
```

Comment:
Translation key should probably be `components.common.shellBox.copy"

Reply:
It is the same as for codebox so I propose to merge it into one key

Reply:
But then if we follow my proposal for CodeBox the key will be `components.shellBox.copy`

Reply:
@ovflowd So with do you think ?

Reply:
`components.common.shellBox.copy` is still the way I'd recommend, as this is its canonical path, and what we're doing on all other component.s

Reply:
Ok i will fix that 😁